test/core/util/test_lb_policies.cc

```diff
@@ -78,6 +78,15 @@ class ForwardingLoadBalancingPolicy : public LoadBalancingPolicy {
   ~ForwardingLoadBalancingPolicy() override = default;
 
   absl::Status UpdateLocked(UpdateArgs args) override {
+    // Use correct config for the delegate load balancing policy
+    auto config =
+        grpc_core::CoreConfiguration::Get()
+            .lb_policy_registry()
+            .ParseLoadBalancingConfig(Json::FromArray({Json::FromObject(
+                {{std::string(delegate_->name()), Json::FromObject({})}})}));
+    if (config.ok()) {
```

Comment:
This should be a `GPR_ASSERT()`.  If it doesn't parse successfully, then we still don't want to pass the parent's config down; we should crash in that case.

Reply:
Done